configserver/src/main/java/com/yahoo/vespa/config/server/tenant/OperatorCertificateSerializer.java

```diff
@@ -0,0 +1,33 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+package com.yahoo.vespa.config.server.tenant;
+
+import com.yahoo.config.model.api.ApplicationRoles;
+import com.yahoo.security.X509CertificateUtils;
+import com.yahoo.slime.Cursor;
+import com.yahoo.slime.Inspector;
+import com.yahoo.slime.Slime;
+import com.yahoo.slime.SlimeUtils;
+
+import java.security.cert.X509Certificate;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OperatorCertificateSerializer {
+
+    public static Slime toSlime(List<X509Certificate> certificateList) {
+        Slime slime = new Slime();
+        Cursor array = slime.setArray();
```

Comment:
You're sure you won't regret making this an array instead of having a top-level object? :)

Reply:
Good point @oyving, we have regretted doing this before.